src/components/Exchange/PositionSeller.js

```diff
@@ -270,19 +340,52 @@ export default function PositionSeller(props) {
       }
     }
 
+    receiveToken = (allowReceiveTokenChange && swapToken) 
+      ? swapToken
+      : collateralToken;
+
     receiveAmount = receiveAmount.add(collateralDelta);
+    receiveAmount = receiveAmount.mul(bigNumberify(1000000));
```

Comment:
may be better to define a constant for 1000000?

Reply:
it's a hardcode for tests)